requirements-rover.txt

```diff
@@ -28,3 +28,4 @@ Twisted==20.3.0
 pytest==3.6.3
 pylint==2.5.3
 yapf==0.30.0
+Jetson.GPIO
```

Comment:
no version?

Reply:
you can just pip freeze to find it I think

Reply:
I would rather keep the version to the latest. It's not like some day they will include breaking changes.

Reply:
(and even if they do, I would rather spend time staying up to date than be stuck on an obsolete version at some point)